aten/src/ATen/native/cpu/UpSampleMoreKernel.cpp

```diff
@@ -332,6 +332,7 @@ void cpu_upsample_linear_channels_last(
     }
   };
 
+  TORCH_INTERNAL_ASSERT(output_slice_size > 0);
```

Comment:
Same here, it's not internal pytorch error, it's user providing invalid data.

Reply:
`upsample_{123}d_shape_check` from [UpSample.h](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/UpSample.h#L59) will check if `width`, `height` and `depth`  are greater than 0, this should guarantee that `output_slice_size`  here is greater than 0. Thus we only need to make sure `channels > 0` for the **channels_last** kernels.